py-polars/tests/unit/test_interop.py

```diff
@@ -605,10 +605,10 @@ def test_from_empty_arrow() -> None:
 
     # 2705
     df1 = pd.DataFrame(columns=["b"], dtype=float)
-    tbl = pa.Table.from_pandas(df1)
+    tbl = pa.Table.from_pandas(df1, preserve_index=True)
```

Comment:
empty dataframes now have a rangeindex by default: https://pandas.pydata.org/docs/dev/whatsnew/v2.0.0.html#empty-dataframes-series-will-now-default-to-have-a-rangeindex

in pyarrow https://arrow.apache.org/docs/python/generated/pyarrow.Table.html :

> The default of None will store the index as a column, except for RangeIndex which is stored as metadata only. Use preserve_index=True to force it to be stored as a column.